docs/tutorial/part-one/index.md

```diff
@@ -378,10 +378,10 @@ case) and you'll see your newly published site! Great work!
 
 In this section you:
 
-- Learned about Gatsby starters, and how to use them to create new projects
-- Learned about JSX syntax
-- Learned about components
-- Learned about Gatsby page components and sub-components
-- Learned about React “props” and reusing React components
+- Learned about Gatsby starters, and how to use them to create new projects.
```

Comment:
These are phrases and not sentences so I would avoid the periods here

(Maybe worth adding to our Style Guide at https://www.gatsbyjs.org/contributing/gatsby-style-guide @gatsbyjs/docs)

Reply:
Actually I thought periods make sense here. But yes your are right these are phrases not sentences .So I will remove periods.